processor/src/tasks/writeKitspaceYaml.ts

```diff
@@ -60,7 +60,8 @@ async function renderKitspaceYamlSummaries(kitspaceYamlArray: Array<KitspaceYaml
   return Promise.all(
     kitspaceYamlArray.map(async yml => ({
       ...yml,
-      summary: await renderSummary(yml.summary),
+      summary: yml.summary,
+      rendered_summary: await renderSummary(yml.summary),
```

Comment:
So this breaks the Zod schema that we eventually want to use on both sides. This is exposing again why we shouldn't be rendering the HTML in the processor. Anyway, just want to vent about it, will merge as is. https://github.com/kitspace/kitspace-v2/issues/594